db/seeds/initial_seed.rb

```diff
@@ -51,3 +48,31 @@
 ].each do |hash|
   NPQCourse.find_or_create_by!(name: hash[:name], id: hash[:id])
 end
+
+all_provider_names = (LeadProvider.pluck(:name) + NPQLeadProvider.pluck(:name)).uniq
+
+all_provider_names.each do |name|
+  CpdLeadProvider.create!(name: name)
+end
```

Comment:
lets' use `find_or_create_by!` please - idempotent seeds are a big convenience.

Reply:
sorry, totally forgot to do this. should be fixed now